### ISSUE TITLE
[Sketcher] Crash when using "Convert Geometry to B-spline" on drag select.

### ISSUE THREAD

**Issue by AjinkyaDahale** [FreeCAD/FreeCAD#6424]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Forums discussion

https://forum.freecadweb.org/viewtopic.php?f=19&t=66470

### Version

0.20 (Development)

### Full version info

```shell
OS: Manjaro Linux (GNOME/gnome)
Word size of FreeCAD: 64-bit
Version: 0.20.27664 (Git)
Build type: Release
Branch: master
Hash: e81eea9bb48af59db92230bbb99f6fbdfef72be8
Python 3.10.2, Qt 5.15.2, Coin 4.0.1, OCC 7.5.3
Locale: English/United States (en_US)
```

### Subproject(s) affected?

Sketcher

### Issue description

Upon using the "Convert Geometry to B-spline" tool on a selection made by pressing and dragging the cursor over multiple sketcher geometries, and subsequently closing sketcher edit mode, sometimes a crash happens.

See forum discussion for example case.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by AjinkyaDahale** [comment]
Addressed in #6403

**Comment by AjinkyaDahale** [comment]
Fixed with 86c91d71a117017db5fc84966ae5aafee1172c2c.
